Zend/zend_vm_execute.h

```diff
@@ -6924,13 +6935,17 @@ static ZEND_VM_COLD ZEND_OPCODE_HANDLER_RET ZEND_FASTCALL ZEND_INIT_METHOD_CALL_
 			} else if (IS_CONST == IS_CV && UNEXPECTED(Z_TYPE_P(function_name) == IS_UNDEF)) {
 				ZVAL_UNDEFINED_OP2();
 				if (UNEXPECTED(EG(exception) != NULL)) {
+					if (!needs_addref) {
 
+					}
```

Comment:
This (and a few lines below) seems pointless? If you need to do something for JIT, maybe put in a fixme comment?

Reply:
This is a generated file, you should mostly ignore its changes.